core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -228,6 +228,16 @@ message ListStatusPOptions {
   optional bool recursive = 4;
   // No data will be transferred.
   optional bool loadMetadataOnly = 5;
+  // For when doing partial listings, an offset to start listing after if included
+  optional int64 offset = 6;
```

Comment:
Offset here is a long which will be the InodeId from where to start the listing. This could be replaced with a path string from where to start the offset.

The advantages of using a long are:
- Fuse uses a long for an offset. So the Fuse client can directly call the Alluxio client api.
- Long is less data to send than a string.
The disadvantages of using a long is:
- If the Inode is deleted in-between calls to the listing, the listing will fail. This is not POSIX compatible.

The advantage of using a string is:
- The file marked by the string can be deleted in-between calls. The listing will just continue with the next file. This is POSIX compatible.

The disadvantage of using a string is:
- The fuse client has to keep some extra state mapping the InodeID to the path during the partial listing.

Any opinions/preferences for this?